src/libs/components/Paragraph/Paragraph.tsx

```diff
@@ -0,0 +1,19 @@
+import { styled } from '@mui/material/styles';
+
+export interface ParagraphProps {
+  margin?: number;
+  fontSize?: number;
+  fontWeight?: number;
+  color?: string;
+}
+
+export const Paragraph = styled('p', {
+  shouldForwardProp: prop =>
+    prop !== 'margin' && prop !== 'fontFamily' && prop !== 'fontSize' && prop !== 'fontWeight' && prop !== 'color',
```

Comment:
fontFamily can be removed here since this style doesn't rely on component's props

Reply:
this is necessary, otherwise there will be an unwanted font